### DISCUSSION TITLE
Support setting SmartQuotes and SmartDashes on iOS

### DISCUSSION THREAD

**Discussion by mark8044** [singerdmx/flutter-quill#572] [Feature Request]
On iOS, there is something called SmartQuotes and SmartDashes which automatically formats quotes and dashes

https://api.flutter.dev/flutter/material/TextField/smartQuotesType.html
https://api.flutter.dev/flutter/material/TextField/smartDashesType.html

These are all enabled by default in iOS textfields and they are in fact enabled on flutter-quill.

On standard flutter TextField you can disable or enable these:
```
                TextField(
                  smartDashesType: SmartDashesType.disabled,
                  smartQuotesType: SmartQuotesType.disabled,
                  controller: _controller,
                ),
```

Could we have the ability to enable/disable this on flutter quill as well?  They are for sure enabled right now out of the box, but there are certain use cases where they need to be disabled

**Comment by mark8044** [comment]
Any update?  This can be a pretty important problem for many people as this gets more use with iOS.

**Reply by mark8044** [reply]
If I can be pointed in the right direction maybe I can figure something out for this?

**Reply by singerdmx** [reply]
![image](https://user-images.githubusercontent.com/122956/150661521-c8e4e2c4-409d-4825-9d5a-1c93a9b5a74a.png)
